crates/miden-lib/src/note/mod.rs

```diff
@@ -13,8 +13,9 @@ use miden_objects::{
 };
 use utils::build_swap_tag;
 
-pub mod scripts;
+mod scripts;
 pub mod utils;
+pub mod well_known_note;
```

Comment:
Looking at this now, we could probably just move the content of `scripts` into the `well_known_notes` module.

Reply:
I addressed it (and the changes in the comment below) in #1206 PR